activities.csv

```diff
@@ -0,0 +1,12 @@
+type,fruit,quantity
+b,banana,20
+b,apple,100
+s,banana,100
+p,banana,13
+r,apple,10
+p,apple,20
+p,banana,5
+s,banana,50
+b,orange,10
+s,orange,5
+p,orange,3
```

Comment:
create in src\main folder resources 
![image](https://github.com/mate-academy/jv-fruit-shop/assets/106664174/af905d0d-f0e7-4cec-9ad0-3ddbeefe3260)